Plotting by universe level

pshriwise opened the pull request:
These changes rely on https://github.com/openmc-dev/openmc/pull/1651 to get the maximum number of universe levels in the geometry. By default, the highest level cell is plotted for a given pixel (the 'all' option). Individual levels can be selected as well.

This is particularly important for visualizing tallies with cell filters containing cells in lower levels of the geometry.

paulromano commented:
When I open up the plotter, the default view is by material. When I switch to by cell, I get an empty plot, and the universe level box is empty. I also see the following at the console:
```
Traceback (most recent call last):
  File "/home/romano/openmc-plotter/openmc_plotter/main_window.py", line 634, in editUniverseLevel
    self.model.activeView.level = int(level)
ValueError: invalid literal for int() with base 10: ''
```
If I then select 'all' for the universe level, the plot shows up, but it looks like you need to change that to be the default when we switch to coloring by cell.

paulromano commented:
Also, this is really cool! It's neat to be able to see the individual universe levels in a model. That was generally something I never did in the old-school plots.xml days, but with the plotter it's really easy.

pshriwise commented:
I really need to remember to try these changes without a `plot_settings.pkl` file present....I updated the default value of the `PlotView` attribute to take care of this. I initially thought I'd want to set it to the max number of levels to start, moved away from that, and then forgot to update the constructor.

Glad you're enjoying it! it does make it easier to interrogate the universe structure of the model for sure.

paulromano commented:
@pshriwise Just wanted to ping you on this. I don't see any updates on the branch here

pshriwise commented:
Made the change but forgot to push! Thanks for the reminder.

paulromano commented:
Thanks @pshriwise!
